exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/main_config_ui.ipynb

```diff
@@ -23,7 +23,8 @@
     "# Need to search for other supporting NBs from the current directory upwards.\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #113.    def get_config_ui() -&gt; widgets.Widget:
</pre><p>name is bit generic</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>